src/apps/routes.clj

```diff
@@ -31,138 +31,138 @@
 
 (defapi app
   {:exceptions cx/exception-handlers}
-  (swagger-ui config/docs-uri
-    :validator-url nil)
-  (swagger-docs
-    {:info {:title "Discovery Environment Apps API"
-            :description "Documentation for the Discovery Environment Apps REST API"
-            :version "2.0.0"}
-     :tags [{:name "service-info", :description "Service Status Information"}
-            {:name "callbacks", :description "General callback endpoints"}
-            {:name "app-categories", :description "App Category endpoints."}
-            {:name "app-hierarchies", :description "App Hierarchy endpoints."}
-            {:name "app-element-types", :description "App Element endpoints."}
-            {:name "apps", :description "App endpoints."}
-            {:name "app-metadata", :description "App Metadata endpoints."}
-            {:name "pipelines", :description "Pipeline endpoints."}
-            {:name "analyses", :description "Analysis endpoints."}
-            {:name "tool-data-containers", :description "Tool Docker Data Container endpoints."}
-            {:name "tools", :description "Tool endpoints."}
-            {:name "workspaces", :description "Workspace endpoints."}
-            {:name "users", :description "User endpoints."}
-            {:name "tool-requests", :description "Tool Request endpoints."}
-            {:name "reference-genomes", :description "Reference Genome endpoints."}
-            {:name "oauth", :description "OAuth callback and information endpoints."}
-            {:name "collaborator-routes", :description "Collaborator Information Routes"}
-            {:name "admin-apps", :description "Admin App endpoints."}
-            {:name "admin-app-metadata", :description "Admin App Metadata endpoints."}
-            {:name "admin-categories", :description "Admin App Category endpoints."}
-            {:name "admin-ontologies", :description "Admin App Ontology endpoints."}
-            {:name "admin-container-images", :description "Admin Tool Docker Images endpoints."}
-            {:name "admin-data-containers", :description "Admin Docker Data Container endpoints."}
-            {:name "admin-tools", :description "Admin Tool endpoints."}
-            {:name "admin-reference-genomes", :description "Admin Reference Genome endpoints."}
-            {:name "admin-tool-requests", :description "Admin Tool Request endpoints."}
-            {:name "admin-oauth", :description "Admin OAuth endpoints."}
-            {:name "admin-integration-data", :description "Admin Integration Data endpoints."}
-            {:name "admin-groups", :description "Admin Group endpoints."}]})
-  (middlewares
+  (swagger-routes
+    {:ui config/docs-uri
+     :options {:ui {:validatorUrl nil}}
+     :data {:info {:title "Discovery Environment Apps API"
+                   :description "Documentation for the Discovery Environment Apps REST API"
+                   :version "2.8.1"}
```

Comment:
Note, I bumped this API version to 2.8.1 since an endpoint was just removed in this version.